public/data/PritamSaha7.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Pritam Saha",
+    "type": "personal",
+    "bio": "I am pritam Saha, Computer Science graduate from Adamas University",
```

Comment:
You might want to capitalize your name here.
```suggestion
    "bio": "I am Pritam Saha, Computer Science graduate from Adamas University",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

Reply:
I have changed the name to capital letter. Is it visible ? please let me know. 
Or should I have to do it from scratch again ?